tests/test_code_gen.py

```diff
@@ -19,85 +18,100 @@
 
 import astor
 
+try:
+    from astunparse_common import AstunparseCommonTestCase
+except ImportError:
+    AstunparseCommonTestCase = None
 
 def canonical(srctxt):
     return textwrap.dedent(srctxt).strip()
 
 
-class CodegenTestCase(unittest.TestCase):
+class Comparisons(object):
+
+    to_source = staticmethod(astor.to_source)
 
-    def assertAstEqual(self, srctxt):
+    assertSrcEqual = unittest.TestCase.assertEqual
+
+    def assertAstEqual(self, ast1, ast2):
+        dmp1 = astor.dump_tree(ast1)
+        dmp2 = astor.dump_tree(ast2)
+        self.assertEqual(dmp1, dmp2)
+
+    def roundtrip_ast(self, srctxt):
         """This asserts that the reconstituted source
            code can be compiled into the exact same AST
            as the original source code.
         """
         srctxt = canonical(srctxt)
         srcast = ast.parse(srctxt)
-        dsttxt = astor.to_source(srcast)
+        dsttxt = self.to_source(srcast)
         dstast = ast.parse(dsttxt)
-        srcdmp = astor.dump_tree(srcast)
-        dstdmp = astor.dump_tree(dstast)
-        self.assertEqual(dstdmp, srcdmp)
+        self.assertAstEqual(srcast, dstast)
 
-    def assertAstEqualIfAtLeastVersion(self, source, min_should_work,
-                                       max_should_error=None):
+    def rtrip_ast_if_vers_gt(self, source, min_should_work,
+                             max_should_error=None):
         if max_should_error is None:
             max_should_error = min_should_work[0], min_should_work[1] - 1
         if sys.version_info >= min_should_work:
-            self.assertAstEqual(source)
+            self.roundtrip_ast(source)
         elif sys.version_info <= max_should_error:
             self.assertRaises(SyntaxError, ast.parse, source)
 
-    def assertAstSourceEqual(self, srctxt):
+    def roundtrip_src(self, srctxt):
         """This asserts that the reconstituted source
            code is identical to the original source code.
-           This is a much stronger statement than assertAstEqual,
+           This is a much stronger statement than roundtrip_ast,
            which may not always be appropriate.
         """
         srctxt = canonical(srctxt)
-        self.assertEqual(astor.to_source(ast.parse(srctxt)).rstrip(), srctxt)
+        self.assertSrcEqual(self.to_source(ast.parse(srctxt)).rstrip(),
+                            srctxt)
 
-    def assertAstSourceEqualIfAtLeastVersion(self, source, min_should_work,
-                                             max_should_error=None):
+    def rtrip_src_if_vers_gt(self, source, min_should_work,
```

Comment:
Same as above. I like the old name because it follows the unittest naming convention and it's clear that it asserts something.